PRINCIPLES.md

```diff
@@ -107,38 +106,38 @@ This library is an experiment and not intended for use. See the
          order to allow rvalues).
        * When the inner type is a template variable, allow const on the inner
          type. The const on the Owning type is unable to be transferred to the
-         inner type, i.e. a const ref<T> can be copied or moved to a non-const
-         ref<T>.
+         inner type, i.e. a const `ref<T>` can be copied or moved to a non-const
+         `ref<T>`.
 1. Class members which have tail padding, or can have based on templates, should
    be marked as [[sus_no_unique_address]] when there are other fields that can
    be packed into the tail padding, if it exists and is usable by the compiler
    implementation.
 1. Bounds are always checked unless you explicitly ask for them to not be.
 1. Lifetimes are always checked unless you explicitly ask for them to not be.
 1. Small headers. C++ compilation speed is proportional to the amount of input.
-    * We will split types into separate headers as much as possible. tokens.
+    * We will split types into separate headers as much as possible.
     * We may provide headers which group together smaller headers for simplicity
       in smaller projects.
     * We will move implementations out of headers, and out of templates, to
       reduce header sizes where possible and where it does not have strong
       performance implications.
-1. We will use doxygen to document all types, functions, and public methods.
+1. We will document all types, functions, and public methods.
     * Documentation will include examples where use is non-trivial.
-    * All functions that can abort() will document under what conditions they do
-      so, with the exception of: use-after-move.
+    * All functions that can a`bort()` will document under what conditions they
+      do so, with the exception of: use-after-move.
 1. No null pointers. Smart pointers have no null state. Functions never return
-    null pointers.
+   null pointers.
 1. No raw pointers by default. Raw pointers throw away all lifetime information,
```

Comment:
i haven't lived up to this one, cuz i decided to try make raw pointers safe instead but yeah just fyi.

Reply:
Ack, that sounds like a separate issue from this PR, heh.

Reply:
yeah definitely :)